### PR TITLE
Fix typo

### PR DESCRIPTION
**Pull request by katrinleinweber**
Same as https://github.com/datacarpentry/scripting-ecology/pull/6 ;-)

### CODE REVIEW

File: lessons/scripting/02-pipeline.md [hunk 1]
```diff
@@ -246,14 +246,14 @@ past. That is, if we put a SQL query into a text file and pipe it to
...
-results as CVS-formatted data. There is one quirk, however. By default
+results as CSV-formatted data. There is one quirk, however. By default
 sqlite does not include a header row in CSV format, but having one
 will help tremendously not only with better documenting our data, but
 also with loading the generated data into other programs such as R. We
 can control this by adding `.headers on` before the query.
 
 Now that we know how to script the database import and SQL subsetting,
-we add the other cvs file imports and replace the query in the sqlite
+we add the other csV file imports and replace the query in the sqlite
```
Comment (hdashnow) [inline lessons/scripting/02-pipeline.md]: capitalisation?

**Comment by hdashnow** [comment]
This is a pretty old workshop. I'm not sure if anyone is still using it?

File: lessons/scripting/02-pipeline.md (hunk 1 above)
Comment (katrinleinweber) [reply lessons/scripting/02-pipeline.md]: Not _exactly_ the same after all :-D I should have copy-pasted. Thanks for noticing!

**Comment by katrinleinweber** [comment]
Maybe not, but I would argue that the question here is: leave a typo in the record or not? ;-)

**Comment by hdashnow** [comment]
Thanks :)

**Comment by katrinleinweber** [comment]
You're welcome, and thanks for merging :-)

**Pull request merged**